src/main/java/com/noticemedan/map/model/MapObjectCreater.java

```diff
@@ -70,36 +78,41 @@ private void initializeBoundsAndMapConstants() {
 		topLeftLat = maxLat * yFactor;
 	}
 
-	private void populateMapObjectEnumMap() {
-		relationList.forEach(this::createAndPutMapObjectToEnumMap);
-		wayList.forEach(this::createAndPutMapObjectToEnumMap);
+	private void populateObjectCollections() {
+		relationList.forEach(this::createAndPutMapObjectToCollection);
+		wayList.forEach(this::createAndPutMapObjectToCollection);
 	}
 
-	private void createAndPutMapObjectToEnumMap(Relation relation) {
+	private void createAndPutMapObjectToCollection(Relation relation) {
 		if (relation == null) return;
 		if (relation.getMembers() == null) return;
 		for (Member member : relation.getMembers()) {
 			if (member.getType().equals("relation"))
-				createAndPutMapObjectToEnumMap(relationMap.get(member.getRef()));
+				createAndPutMapObjectToCollection(relationMap.get(member.getRef()));
 			if (member.getType().equals("way"))
-				createAndPutMapObjectToEnumMap(wayMap.get(member.getRef()));
+				createAndPutMapObjectToCollection(wayMap.get(member.getRef()));
 		}
 	}
 
-	private void createAndPutMapObjectToEnumMap(Way way) {
+	private void createAndPutMapObjectToCollection(Way way) {
 		if (way == null) return;
 		if (way.getNodeRefs() == null) return;
 		OSMType type = OSMType.UNKNOWN;
-		if (way.getTags() != null) {
+		if (way.getTags() != null)
```

Comment:
Hvordan kører den uden {

Reply:
Den kører fint uden, det er for at ease øjnene lidt i en allerede super-forvirrende method ;)